lib/data/prisoner_packs.yml

```diff
@@ -368,6 +366,8 @@
   pdf: /government/publications/nigeria-prisoner-pack
   lawyer: /government/publications/nigeria-list-of-lawyers
 - slug: north-korea
+- slug: north-macedonia
+  pdf: /government/publications/macedonia-prisoner-pack
```

Comment:
Looks like `government/publications/north-macedonia-prisoner-pack` is already up?

Reply:
@deborahchua and I think that content designers already made those changes, so we'll update them here. 👍